docs/getting_started/tasks_and_workflows.md

```diff
@@ -83,7 +83,7 @@ invaluable for catching bugs early.
 
 Flyte's type system is also used for caching, data lineage tracking, and
 automatic serialization and deserialization of data as it's passed from one task
-to another. You can learn more about it in the {ref}`User Guide <flyte_type_system>`.
+to another. You can learn more about it in the {ref}`User Guide <data_types_and_io>`.
```

Comment:
revert ref to `flyte_type_system` to avoid breaking changes with other docs sites